File: 5-kyu/Simple Pig Latin.py
def pig_it(text):
    arr = text.split(' ')
    result = []
    for word in arr:
        if word.isalpha():
            if len(word) > 1:
                result.append(word[1:]+word[0]+'ay')
            else:
                result.append(word + 'ay')
        else:
            result.append(word)
    return ' '.join(result)

File: 5-kyu/Simple Pig Latin.js
function pigIt(str){
  let result = [];
  let arr = str.split(" ");
  for (let i=0; i<arr.length; i += 1) {
    if (arr[i]!='?' && arr[i]!='!') {
      let word = arr[i];
      let letter = word.charAt(0);
      let newWord = word.slice(1) + letter + "ay";
      result.push(newWord);
    } else {
      result.push(arr[i]);
    }
  }
  return result.join(" ");
}
